modules/syslogformat/tests/Makefile.am

```diff
@@ -0,0 +1,13 @@
+modules_syslogformat_tests_TESTS		= \
+	modules/syslogformat/tests/test_parse_timestamp
+
+check_PROGRAMS				+= ${modules_syslogformat_tests_TESTS}
+
+modules_syslogformat_tests_test_parse_timestamp_SOURCES	= modules/syslogformat/tests/test_parse_timestamp.c
+modules_syslogformat_tests_test_parse_timestamp_CFLAGS	= $(TEST_CFLAGS) -I$(top_srcdir)/modules/syslogformat
+modules_syslogformat_tests_test_parse_timestamp_LDFLAGS	= $(LDFLAGS_FORCE_FLAT_NAMESPACE)
```

Comment:
Have you considered adding `force-flat-namespace` for all of the unit tests just like in `cmake` version ?